ddtrace/opentracer/tracer.go

```diff
@@ -84,3 +86,12 @@ func (t *opentracer) Extract(format interface{}, carrier interface{}) (opentraci
 		return nil, opentracing.ErrUnsupportedFormat
 	}
 }
+
+func (t *opentracer) ContextWithSpanHook(ctx context.Context, openSpan opentracing.Span) context.Context {
+	ddSpan, ok := openSpan.(*span)
+	if !ok {
+		return ctx // unchanged
```

Comment:
I think the comment is unnecessary.
```suggestion
		return ctx
```